cmd/port-layer-server/main.go

```diff
@@ -98,6 +99,22 @@ func main() {
 		extraconfig.SetLogLevel(log.DebugLevel)
 	}
 
+	if vchConfig.Diagnostics.DebugLevel > 4 {
```

Comment:
This is here solely to allow capture of the raw soap requests issued to vSphere.
This may be a useful capability to keep, but likely needs tidying up somewhat.